src/Compilers/Core/Portable/Syntax/SyntaxList.SeparatedWithManyChildren.cs

```diff
@@ -10,24 +12,24 @@ internal class SeparatedWithManyChildren : SyntaxList
         {
             private readonly ArrayElement<SyntaxNode>[] _children;
 
-            internal SeparatedWithManyChildren(InternalSyntax.SyntaxList green, SyntaxNode parent, int position)
+            internal SeparatedWithManyChildren(InternalSyntax.SyntaxList green, SyntaxNode? parent, int position)
                 : base(green, parent, position)
             {
                 _children = new ArrayElement<SyntaxNode>[(green.SlotCount + 1) >> 1];
             }
 
-            internal override SyntaxNode GetNodeSlot(int i)
+            internal override SyntaxNode? GetNodeSlot(int i)
             {
                 if ((i & 1) != 0)
                 {
                     //separator
                     return null;
                 }
 
-                return this.GetRedElement(ref _children[i >> 1].Value, i);
+                return this.GetRedElement(ref _children[i >> 1].Value!, i);
```

Comment:
❔ Why this suppression?

Reply:
The parameter here expects a nullable input but this is non-null